video_xblock/backends/html5.py

```diff
@@ -86,3 +86,12 @@ def player_data_setup(context):
             "playbackRates": [0.5, 1, 1.5, 2],
         })
         return result
+
+    @property
+    def trans_fields(self):
+        """
+        List of VideoXBlock fields to display on `Manual & default transcripts` panel for Html5 Player.
+        """
+        fields = ['transcripts', 'default_transcripts']
```

Comment:
I think it would be better to replace it with the parent property
```suggestion
        fields = super().trans_fields
```

Reply:
done